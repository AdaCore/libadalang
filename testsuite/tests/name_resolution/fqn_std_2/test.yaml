driver: name-resolution
input_sources: [teststd.ads]
