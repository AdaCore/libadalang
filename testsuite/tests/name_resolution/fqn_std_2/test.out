Analyzing teststd.ads
#####################

Resolving xrefs for node <ConcreteTypeDecl ["My_Int"] teststd.ads:2:4-2:40>
***************************************************************************

Expr: <DottedName teststd.ads:2:23-2:39>
  references:    <DefiningName "Integer" __standard:4:8-4:15>
  type:          None
  expected type: None
Expr: <Id "Standard" teststd.ads:2:23-2:31>
  references:    <DefiningName "Standard" __standard:1:9-1:17>
  type:          None
  expected type: None
Expr: <Id "Integer" teststd.ads:2:32-2:39>
  references:    <DefiningName "Integer" __standard:4:8-4:15>
  type:          None
  expected type: None

Resolving xrefs for node <ObjectDecl ["A"] teststd.ads:3:4-3:38>
****************************************************************

Expr: <DottedName teststd.ads:3:8-3:31>
  references:    <DefiningName "My_Int" teststd.ads:2:9-2:15>
  type:          None
  expected type: None
Expr: <DottedName teststd.ads:3:8-3:24>
  references:    <DefiningName "Teststd" teststd.ads:1:9-1:16>
  type:          None
  expected type: None
Expr: <Id "Standard" teststd.ads:3:8-3:16>
  references:    <DefiningName "Standard" __standard:1:9-1:17>
  type:          None
  expected type: None
Expr: <Id "Teststd" teststd.ads:3:17-3:24>
  references:    <DefiningName "Teststd" teststd.ads:1:9-1:16>
  type:          None
  expected type: None
Expr: <Id "My_Int" teststd.ads:3:25-3:31>
  references:    <DefiningName "My_Int" teststd.ads:2:9-2:15>
  type:          None
  expected type: None
Expr: <Int teststd.ads:3:35-3:37>
  references:    None
  type:          <ConcreteTypeDecl ["Universal_Int_Type_"] __standard:116:3-116:45>
  expected type: <ConcreteTypeDecl ["My_Int"] teststd.ads:2:4-2:40>

Resolving xrefs for node <EndName teststd.ads:4:5-4:12>
*******************************************************

Expr: <EndName teststd.ads:4:5-4:12>
  references:    <DefiningName "Teststd" teststd.ads:1:9-1:16>
  type:          None
  expected type: None
Expr: <Id "Teststd" teststd.ads:4:5-4:12>
  references:    <DefiningName "Teststd" teststd.ads:1:9-1:16>
  type:          None
  expected type: None


Done.
